### PR TITLE
Add notifications chapter

### PR DESCRIPTION
**Pull request by eduardoj**
To review this PR, generate the HTML version of the OBS user guide with:

```
docker-compose run --rm obs-docu daps -d DC-obs-user-guide html
```

Then, with open the `Using the OBS Web UI` page at the `Notifications` section in your browser:
```
firefox build/obs-user-guide/html/book.obs-user/cha.obs.notifications.html
```

### CODE REVIEW

File: xml/obs_best_practice_notifications.xml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Notifications</title>
+ <para>
+  The "Notifications" page is the place that keeps you up to date with your
+  day-to-day OBS work.
+ </para>
+ <para>
+  You can configure them in detail to receive just what is from your interest.
```
Comment (krauselukas) [inline xml/obs_best_practice_notifications.xml:17]:
```suggestion
  You can configure them in detail to receive just what is of your interest.
```

File: xml/obs_best_practice_notifications.xml [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+  Not only that, with the web UI interface, you can easily filter your
+  notifications and mark them as read, improving your OBS workflow experience.
+ </para>
+
+ <sect2 xml:id="cha.obs.best-practices.webuiusage.notifications_configuration">
+  <title>Notifications Configuration</title>
+  <para>
+   Click on the "Change Your Notifications" link to define what notifications
```
Comment (krauselukas) [inline xml/obs_best_practice_notifications.xml:25]:
```suggestion
   Click on the "Change Your Notifications" link to define which notifications
```

File: xml/obs_best_practice_notifications.xml [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+   you receive. You can see this link on your Profile page in the "Actions"
```
Comment (krauselukas) [inline xml/obs_best_practice_notifications.xml:26]:
```suggestion
   you want to receive. You can see this link on your Profile page in the "Actions"
```

File: xml/obs_best_practice_notifications.xml (hunk 2 above)
Comment (krauselukas) [reply xml/obs_best_practice_notifications.xml:25]: Well, `what` is fine in this case I guess

Comment (eduardoj) [reply xml/obs_best_practice_notifications.xml]: Both `what` and `which` works, I guess. Also applied.

File: xml/obs_notifications.xml [hunk 4]
```diff
@@ -300,4 +300,50 @@
...
+   </para>
+   <screen>osc api -X GET "/my/notifications/3"</screen>
+   <para>
+     The command will return the content of that notification.
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;status code="anonymous_user"&gt;
+    &lt;summary&gt;Anonymous user is not allowed here - please login&lt;/summary&gt;
+&lt;/status&gt;
```
Comment (eduardoj) [inline xml/obs_notifications.xml:343]: That's an "Anonymous user" error reponse...

Comment (danidoni) [reply xml/obs_notifications.xml:343]: Yes, I'm trying to figure out why it is returning this, even when I'm logged in...

File: xml/obs_notifications.xml [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+        &lt;who&gt;User 2&lt;/who&gt;
+        &lt;event_type&gt;comment_for_package&lt;/event_type&gt;
+        &lt;when/&gt;
+    &lt;/notification&gt;
+&lt;/notifications&gt;
+   </screen>
+   <para>
+     To check the contents of a single notification, query the following endpoint:
+   </para>
+   <screen>osc api -X GET "/my/notifications/3"</screen>
```
Comment (krauselukas) [inline xml/obs_notifications.xml:335]: We dont have this implemented, there is only an `index` and `update` endpoint

**Review by krauselukas — CHANGES_REQUESTED** [review]

File: xml/obs_notifications.xml [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+     It will say if the operation went well or not.
+   </para>
+   <screen language="xml">
+&lt;?xml version="1.0" encoding="UTF-8"?&gt;
+&lt;status code="ok"&gt;
+    &lt;summary&gt;Ok&lt;/summary&gt;
+&lt;/status&gt;
+   </screen>
+   <para>
+     To check required parameters and options, please check our throughly documented <link xlink:href="https://build.opensuse.org/apidocs-new/#/Notifications/get_my_notifications">Experimental API Documentation</link>.
```
Comment (krauselukas) [inline xml/obs_notifications.xml:346]:
```suggestion
     To check required parameters and options, please check our thoroughly documented <link xlink:href="https://build.opensuse.org/apidocs-new/#/Notifications/get_my_notifications">Experimental API Documentation</link>.
```

Comment (krauselukas) [reply xml/obs_notifications.xml:346]: @danidoni I wouldn't call it experimental, I think its already a bit more then that :) also you twice use the word `check`

Maybe something like: `For more information regarding the notifications API endpoints, checkout the API Documentation

File: xml/obs_notifications.xml [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+
+<chapter version="5.1" xml:id="cha.obs.notifications"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Notifications</title>
+
+ <note>
+  <title>
+   The feature described in this chapter is not included yet in any OBS
```
Comment (krauselukas) [inline xml/obs_notifications.xml:15]:
```suggestion
   The feature described in this chapter is not included yet in any official OBS release
```

File: xml/obs_notifications.xml [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
+   official release.
```
Comment (krauselukas) [inline xml/obs_notifications.xml:16]:
```suggestion

```

File: xml/obs_notifications.xml [hunk 9]
```diff
@@ -0,0 +1,349 @@
...
+  </title>
+
+  <para>
+   That means this feature is deployed in the
```
Comment (krauselukas) [inline xml/obs_notifications.xml:20]: @eduardoj how about mentioning its only available through our Unstable project which we deploy on `build.opensuse.org`

File: xml/obs_notifications.xml (hunk 6 above)
Comment (krauselukas) [reply xml/obs_notifications.xml:346]: or maybe `checkout the developer documentation`

Comment (eduardoj) [reply xml/obs_notifications.xml:346]: > @danidoni I wouldn't call it experimental, I think its already a bit more then that :)

Sure, but the "official API documentation" is still another one. Mentioning only "API documentation" doesn't include the idea of a not yet officially approved documentation. I would keep the sentence with "Experimental API Documentation".

Comment (krauselukas) [reply xml/obs_notifications.xml:346]: okay, makes sense, would still rephrase it a bit, just to make the sentence a bit nicer, maybe `For more information regarding the notifications API endpoints, checkout the Experimental API Documentation` ?

File: xml/obs_notifications.xml (hunk 9 above)
Comment (eduardoj) [reply xml/obs_notifications.xml]: Fine. Applied.

File: xml/obs_notifications.xml [hunk 10]
```diff
@@ -0,0 +1,350 @@
...
+<chapter version="5.1" xml:id="cha.obs.notifications"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+ <title>Notifications</title>
+
+ <note>
+  <title>
+   The feature described in this chapter is not included yet in any OBS
+   official release.
```
Comment (krauselukas) [inline xml/obs_notifications.xml:16]:
```suggestion
   The feature described in this chapter is not included yet in any official OBS
   release.
```

**Review by danidoni — APPROVED** [review]

**Pull request merged**